src/__tests__/index.js

```diff
@@ -372,6 +372,107 @@ test('should receive inner ref if specified', () => {
   expect(getRef).toHaveBeenCalled()
 })
 
+test('should pass inner ref to underlying component if forwarded', () => {
+  let nodeType = null
+  const getRef = n => (nodeType = n)
+
+  class InnerComp extends React.Component {
+    render() {
+      return (
+        <div>
+          <span ref={this.props.innerRef} />
+        </div>
+      )
+    }
+  }
+
+  const Comp = glamorous(InnerComp, {forwardProps: ['innerRef']})({
+    marginLeft: '24px',
+  })
+
+  mount(<Comp innerRef={getRef} blue />)
+
+  expect(nodeType).toBeInstanceOf(HTMLElement)
+  expect(nodeType.tagName).toBe('SPAN')
+})
+
+test('should not pass inner ref to underlying component if not forwarded', () => {
+  let nodeType = null
+  const getRef = n => (nodeType = n)
+
+  class InnerComp extends React.Component {
+    render() {
+      return (
+        <div>
+          <span ref={this.props.innerRef} />
+        </div>
+      )
+    }
+  }
+
+  const Comp = glamorous(InnerComp)({
+    marginLeft: '24px',
+  })
+
+  mount(<Comp innerRef={getRef} />)
+
+  expect(nodeType).toBeInstanceOf(InnerComp)
+})
+
+test('should pass inner ref to underlying component if forwarded and rootEl is set', () => {
+  let nodeType = null
+  const getRef = n => (nodeType = n)
+
+  class InnerComp extends React.Component {
+    render() {
+      return (
+        <div>
+          <span ref={this.props.innerRef} />
+        </div>
+      )
+    }
+  }
+
+  const Comp = glamorous(InnerComp, {
+    rootEl: 'span',
```

Comment:
It doesn't technically matter, but `rootEl` is actually a `div` in this case. Should probably fix it just to avoid confusing future contributors.

Reply:
well, it kind of depends where we pass props received from glamorous, isn't it?  In this case we pass them neither on `div` nor `span`. :thinking:

Reply:
fixed 👍